### PR TITLE
Constant circle speed

### PR DESCRIPTION
**Pull request by YoNeXia**
During test, circle speed of each circle was different. I could see the difference of speed particulary when my gaze go inside the circle or not.
So I check the code and the problem are from script `CircleLife.cs`. When program is running, before apply an effect on red circle, speed of circle was reset.

Now, circle speed is always the same one, is constant. To do this, just need to delete the reset of circle speed.

This pull request resolve the issur #19 .

### CODE REVIEW

**Review by QuentinDaveau — APPROVED** [review]
It was really hard to follow with all the changes, but I think everything is good.

**Comment by YoNeXia** [comment]
@bastianilso If you want to try it again as the headset is setup now. 😄
